mmfunctions

```diff
@@ -174,6 +175,15 @@ _premis_event_list(){
     fi
 }
 
+_generate_fingerprint(){
+    if [ ${PERCEPTUAL_HASH} = "1" ] ; then
+        SIGNATURE="$(basename ${INPUT%.*})_signature.xml"
+        ~/ffmpeg/ffmpeg -i "${INPUT}" -vf signature=format=xml:filename="${OUTDIR_INGESTFILE}/${MEDIAID}/metadata/fileMeta/objects/${SIGNATURE}" -map 0:v -f null -
```

Comment:
Recommend the `filename=` writes the file outside of `fileMeta` but within `metadata`.